release notes/v0.55.0.md

```diff
@@ -0,0 +1,205 @@
+k6 `v0.55.0` is here 🎉! This release includes:
+
+- ⚠️ The deprecated StatsD output has been removed.
+- ⚠️ The experimental `k6/experimental/tracing` module has been removed.
+- URL grouping support in the browser module.
+- Top-level `await` support.
+- Complete RSA support for `k6/experimental/webcrypto`.
+
+## Breaking changes
+
+### `k6/experimental/tracing` module removed [`#3855`](https://github.com/grafana/k6/pull/3855)
+
+The experimental `k6/experimental/tracing` module has been removed, in favor of a replacement [jslib](https://jslib.k6.io/http-instrumentation-tempo/), please look at the [docs on how to transition](https://grafana.com/docs/k6/latest/javascript-api/jslib/http-instrumentation-tempo/#migration-from-k6experimentaltracing), [`#3855`](https://github.com/grafana/k6/pull/3855).
```

Comment:
"https://jslib.k6.io/http-instrumentation-tempo/" -- That links points to the old k6.io site, and it's currently 404ing. Does the URL need to be updated?

Reply:
```suggestion
The experimental `k6/experimental/tracing` module has been removed, in favor of a replacement [jslib](https://jslib.k6.io/http-instrumentation-tempo/) polyfill, please look at the [docs on how to transition](https://grafana.com/docs/k6/latest/javascript-api/jslib/http-instrumentation-tempo/#migration-from-k6experimentaltracing), [`#3855`](https://github.com/grafana/k6/pull/3855).
```

Is it being replaced with a polyfill?